extensions/ql-vscode/src/remote-queries/remote-queries-manager.ts

```diff
@@ -317,10 +318,12 @@ export class RemoteQueriesManager extends DisposableObject {
     if (resultIndex) {
       const metadata = await this.getRepositoriesMetadata(resultIndex, credentials);
       const queryResult = this.mapQueryResult(executionEndTime, resultIndex, queryId, metadata);
+      const resultCount = queryResult.analysisSummaries.reduce((acc, cur) => acc + cur.resultCount, 0);
```

Comment:
Do we have this available in one of our helper functions somewhere now (or is it in a PR)?

Reply:
We don't! Ignore me 😄

Reply:
It was a good idea so I've extracted a method for that: https://github.com/github/vscode-codeql/pull/1432/commits/9078f411b9d112ad9887ed37c5daac69a72b5b15